### DISCUSSION TITLE
Source "@openzeppelin/contracts/token/ERC721/ERC721.sol" not found: File import callback not supported

### DISCUSSION THREAD

**Discussion by gregfraz** [ConsenSys-archive/truffle#4831] [General Q&A]
I know this issue is closed, but this is still a problem:
Source "@openzeppelin/contracts/token/ERC721/ERC721.sol" not found: File import callback not supported.

![truffleSyntaxError02](https://user-images.githubusercontent.com/24599002/157538580-449014ac-db56-4eef-b72e-404c4ea584d0.png)

![truffleSyntaxError03](https://user-images.githubusercontent.com/24599002/157539098-7ab74177-62ad-494b-ac26-0c4378254224.png)

Is there a work-around for this?

**Comment by fletch-r** [comment]
Download the "Solidity + Hardhat" VSCode extention and make sure you've done
`npm i -D @openzeppelin/contracts`
